src/showingpreviously/cinemas/cineworld.py

```diff
@@ -0,0 +1,105 @@
+import json
+
+from datetime import datetime, timedelta
+import showingpreviously.requests as requests
+from showingpreviously.model import ChainArchiver, CinemaArchiverException, Chain, Cinema, Screen, Film, Showing
+from showingpreviously.consts import STANDARD_DAYS_AHEAD, UK_TIMEZONE
```

Comment:
This is good, and we should make sure all the existing archivers use these constants.

Reply:
They do